docs/superpowers/plans/2026-06-08-eating-corpses.md

```
@@ -0,0 +1,728 @@
+
+**Step 2: Give monsters corpses.** In `data/monsters.toml`, add to `[monster.rat]`:
+```toml
+corpse = "rat_corpse"
+```
+and to `[monster.ghoul]`:
+```toml
+corpse = "ghoul_corpse"
+```
+
+**Step 3: Ignore the morgue artifact.** Append to repo-root `/Users/arda/projects/tsl/.gitignore`:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)